guides/source/configuring.md

```diff
@@ -1410,7 +1410,20 @@ development:
 
 By default Rails ships with three environments: "development", "test", and "production". While these are sufficient for most use cases, there are circumstances when you want more environments.
 
-Imagine you have a server which mirrors the production environment but is only used for testing. Such a server is commonly called a "staging server". To define an environment called "staging" for this server, just create a file called `config/environments/staging.rb`. Please use the contents of any existing file in `config/environments` as a starting point and make the necessary changes from there.
+Imagine you have a server which mirrors the production environment but is only used for testing. Such a server is commonly called a "staging server". To create this environment the following actions are required:
```

Comment:
```suggestion
Imagine you have a server mirroring the production environment, but is only used for testing. Such a server is commonly called a "staging server". To create a staging environment perform the following actions:
```
I added a comma, but I also added some minor wording suggestions. What do you think?